controllers/openstackcluster_controller.go

```diff
@@ -294,15 +286,17 @@ func reconcileBastion(log logr.Logger, osProviderClient *gophercloud.ProviderCli
 	log.Info("Reconciling Bastion")
 
 	if openStackCluster.Spec.Bastion == nil || !openStackCluster.Spec.Bastion.Enabled {
-		return deleteBastion(log, osProviderClient, clientOpts, openStackCluster)
+		return deleteBastion(log, osProviderClient, clientOpts, cluster, openStackCluster)
 	}
 
 	computeService, err := compute.NewService(osProviderClient, clientOpts, log)
 	if err != nil {
 		return err
 	}
 
-	instance, err := computeService.InstanceExists(fmt.Sprintf("%s-bastion", cluster.Name))
+	clusterName := fmt.Sprintf("%s-%s", cluster.Namespace, cluster.Name)
+
+	instance, err := computeService.InstanceExists(getBastionName(clusterName))
```

Comment:
This changes the name of the bastion host right? I think that's okay for v1alpha3 => v1alpha4, but we should document this. Can you maybe start a MD to collect "breaking" changes for v1alpha4 so we can post them with our release notes when we release?

Reply:
Yes, but I should have focused on fixing deleting. I will update the name back.

Reply:
Okay thx